p/api/greader.php

```diff
@@ -481,7 +481,11 @@ function entriesToArray($entries) {
 	}
 
 	$items = array();
-	foreach ($entries as $entry) {
+	foreach ($entries as $item) {
+		$entry = Minz_ExtensionManager::callHook('entry_before_display', $item);
```

Comment:
More of a note for other people reading this, because at least at present it doesn't seem too important, but in the issue there was the suggestion of differentiating between `entry_before_display` and `entry_before_api` (or something along those lines).

> Perhaps it can be an option for extensions to tell where it need to be enabled ? (WebUI, greader, etc.)

Reply:
I have no objections to distinguish `entry_before_display` and `entry_before_api` (maybe different naming like `entry_before_api_display`?) but I would stick with current solution for now. It makes the PR easier and we'll be able to merge it quicker :)